ui-components/src/navigation/TabNav.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { LocalizedLink } from "../actions/LocalizedLink"
+import "./TabNav.scss"
+
+export interface TabProps {
+  href: string
+  current?: boolean
+  children: React.ReactNode
+}
+
+const Tab = (props: TabProps) => {
+  return (
+    <li className="tab-nav__tab">
+      <LocalizedLink
+        className={props.current ? "is-active" : undefined}
```

Comment:
I think we can simplify it to just: `props.current && "is-active"`

Reply:
Well that only works on the "true" condition…on the false condition the expression literally returns false, rather than undefined